### ISSUE TITLE
All conjunctions are CCONJ

### ISSUE THREAD

**Issue by ceramisch** [UniversalDependencies/UD_Portuguese-GSD#15]
I noticed that the corpus does not contain any `SCONJ`, only CCONJ.
Well, at least the obvious subordinative conjunctions such as _que_, _quando_, _onde_, _porque_ etc. could be (automatically?) converted to SCONJ instead of CCONJ.

The easy fix could simply consist of:
```python
if token["deprel"] == "mark" : token["upos"] = "SCONJ"
``` 

Any corner cases or objections to automatic correction?

**Comment by arademaker** [comment]
After running the following query in the Bosque documents, I confimed that only SCONJ was returned from tokens with deprel mark:

```
% awk '$4 ~ /^(SCONJ|CCONJ)$/ {print $4,$2,$8}' *.conllu | sort | uniq -c  | grep mark
```

On the other hand, not only `SCONJ` can have deprel `mark`:

```
% awk '$8 ~ /^mark$/ {print $4}' *.conllu | sort | uniq -c
  56 ADP
 169 ADV
  10 DET
5165 SCONJ
   3 VERB
```

The verb cases can be mistakes. But the ADP and ADV can be related to MWEs.

**Comment by martinpopel** [comment]
Yes, `if node.upos == 'CCONJ' and node.deprel == 'mark': node.upos = 'SCONJ'` should be a safe rule. It seems I [introduced](https://github.com/udapi/udapi-python/commit/93b9cd4d6af515c239a1b66793a968aecac829c3) it only after someone [converted UD_Portuguese-GSD to UDv1](https://github.com/UniversalDependencies/UD_Portuguese-GSD/commit/06eed57ae3de0f7589c0762c6a2fb2787d241907#diff-774101019fa37ce5e982e41353e9415dR19). There may be [other fixes](https://github.com/udapi/udapi-python/blob/master/udapi/block/ud/exgoogle2ud.py) useful for former Google Universal Dependency Treebanks.

**Comment by arademaker** [comment]
hi @martinpopel , thank you for pointing to the history of changes and the possible useful code. Yes, maybe we can try your code and see if the changes make sense. What do you think @ceramisch ?

**Comment by martinpopel** [comment]
Yes, you can try the code, but you should inspect the changes (and code). Let me know if you need any help with Udapi.

Now, I realize I never touched UD_Portuguese-GSD. It was [UD_Portuguese-PUD](https://github.com/UniversalDependencies/UD_Portuguese-PUD), which I converted from the Google annotation style. BTW: UD_Portuguese-PUD contains [no SCONJ](https://github.com/UniversalDependencies/UD_Portuguese-PUD/blob/master/stats.xml#L22), but also no `CCONJ&mark` (most "*que*" are annotated as PRON or ADP). Obviously, this is just one of the many inconsistencies between UD_Portuguese-PUD and UD_Portuguese-GSD and UD_Portuguese-Bosque.

**Comment by dan-zeman** [comment]
I think it was me who converted UD_Portuguese-GSD from Google Stanford Dependencies to Universal Dependencies. But if the original had only one type of conjunction, I probably kept it as it was. Feel free to improve it.

**Comment by arademaker** [comment]
Thank you Dan, this corpus still contains many problems. I will try to work more on it for the next release.
